tests/acceptance/view-only-link-test.ts

```diff
@@ -68,8 +68,8 @@ module('Acceptance | view-only-links', hooks => {
         server.create('root');
 
         const viewOnlyToken = 'thisisatoken';
-        await visit(`/support?view_only=${viewOnlyToken}`);
-        assert.equal(currentURL(), `/support?view_only=${viewOnlyToken}`);
+        await visit(`/settings/profile/name?view_only=${viewOnlyToken}`);
```

Comment:
You're getting a timeout error on these. Maybe do `/settings/account` instead, since the profile stuff isn't really implemented?

Reply:
Sorry, my brain was not working when I suggested settings. I was trying to pick a non-project/registration page that would work when logged in and when logged out, but the settings pages obviously only work when logged in. The registries discover page looks like the one to use, as it will work in both states and does show the banner. Sorry for the bad advice.